### PR TITLE
feat: update favicon and og

### PR DESCRIPTION
**Pull request by alexiscolin**
This PR updates favicon and og images / info.

- More favicon formats and sizes (for quality, compatibility and HD matters)
- All images have been optimized to reduce their sizes
- Adds a default `og` image to improve UX and SEO when users share the website
- Completes basic `og` info such as `title` and `description`

Note:
_Would be cool to use a themed (light/dark) favicon by using native `prefered-color-scheme` in the next `gnoweb` iteration (require to update the light/dark theme system a bit)_

_We could also add a `webmanifest.json`to give more details to android devices (when users install the website as a webapp and/or pin it to their homescreen). Same for IE with a `msImage` and a browserconfig.xml file_

### CODE REVIEW

**Review by zivkovicmilos — APPROVED** [review]

File: gno.land/pkg/gnoweb/views/funcs.html [hunk 1]
```diff
@@ -82,6 +82,16 @@
...
+<link rel="apple-touch-icon" sizes="180x180" href="/static/img/apple-touch-icon.png" />
+<link rel="icon" type="image/png" sizes="32x32" href="/static/img/favicon-32x32.png" />
+<link rel="icon" type="image/png" sizes="16x16" href="/static/img/favicon-16x16.png" />
+<link rel="mask-icon" href="/static/img/safari-pinned-tab.svg" color="#6f6f6f" />
+<meta name="theme-color" content="#ffffff" />
+<meta property="og:title" content="Gno.land - {{.Data.RealmName}}" />
+<meta property="og:url" content="https://gno.land" />
+<meta property="og:image" content="https://gno.land/static/img/og.png" />
+<meta property="og:site_name" content="gno.land" />
+<meta property="og:description" content="{{ .Data.Description }}" />
```
Comment (zivkovicmilos) [inline gno.land/pkg/gnoweb/views/funcs.html:94]: Where is this `Description` defined?

**Review by moul — APPROVED** [review]

File: gno.land/pkg/gnoweb/views/funcs.html (hunk 1 above)
Comment (alexiscolin) [reply gno.land/pkg/gnoweb/views/funcs.html:94]: To be honest, I reused what was already used for description metadata and hadn't tested it yet. But you are right, the content is always empty... Also, in addition to the generic default desc, we should be able (ideally) to have a dynamic desc.

Comment (alexiscolin) [reply gno.land/pkg/gnoweb/views/funcs.html:94]: What should we write as generic description. Any idea @leohhhn @michelleellen ?

Comment (moul) [reply gno.land/pkg/gnoweb/views/funcs.html:94]: Let it empty for now.

**Review by moul — APPROVED** [review]

**Pull request merged**